### PR TITLE
MCLOUD-6120: Crypt key absence message appear on installation process

### PR DESCRIPTION
**Pull request by NadiyaS**
### Description

During installation of Magento `crypt key` is generated automatically.
Now `cloud.log` contains notice that this key is absent even when Magento was installed:
```    
    [2020-06-02 14:10:08] NOTICE: Crypt key missing. Add the crypt key to the "app/etc/env.php" file, or set the "CRYPT_KEY" environment variable in the Cloud Project web UI.  
    [2020-06-02 14:10:08] INFO: Checking if db exists and has tables  
    [2020-06-02 14:10:08] NOTICE: Starting install.  
    [2020-06-02 14:10:08] INFO: Installing Magento. 
```

Need to remove that message even for upgrade process as it duplicates after implementation https://github.com/magento/ece-tools/pull/732

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-6120

### Manual testing scenarios

1. Uninstall Magento and run redeployment
**Expected result:** magento was installed. cloud.log does not have notice that crypt key does not exist
2. on installed Magento remove crypt/key from app/etc/env.php file and run redeployment
**Expected result:** deployment was failed with error that crypt key does not exist.
3. Add CRYPT_KEY cloud environment variable (and run redeployment)
**Expected result:** Magento upgraded successfully
4. Uninstall Magento and run redeployment (CRYPT_KEY exists from prev. step)
**Expected result:** magento was installed. env.php file has `crypt/key` the same as in `CRYPT_KEY` variable
5. Remove CRYPT_KEY cloud environment variable (and run redeployment)
**Expected result:** Magento upgraded successfully

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: src/Step/Deploy/InstallUpdate/ConfigUpdate/SetCryptKey.php [hunk 1]
```diff
@@ -73,11 +73,6 @@ public function execute(): void
         $key = $this->environment->getCryptKey();
 
         if (empty($key)) {
-            $this->logger->notice(
```
Comment (shiftedreality) [inline src/Step/Deploy/InstallUpdate/ConfigUpdate/SetCryptKey.php]: Why do you remove the message?

Comment (NadiyaS) [reply src/Step/Deploy/InstallUpdate/ConfigUpdate/SetCryptKey.php]: equal message presents in `install-update` step and only for upgrade case https://github.com/magento/ece-tools/blob/develop/src/Config/State.php#L95
where it is not only notice, it is exception

**Review by oshmyheliuk — DISMISSED** [review]

**Review by shiftedreality — DISMISSED** [review]

**Review by VladimirZaets — DISMISSED** [review]

**Review by oshmyheliuk — DISMISSED** [review]

**Comment by mveeramneni** [comment]
Marking on hold as Nadiia's is figuring out why value is not set

**Review by oshmyheliuk — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by mveeramneni** [comment]
QA Approved

**Pull request merged**